src/app/components/age-selector/age-selector.component.ts

```diff
@@ -57,7 +60,8 @@ export class AgeSelectorComponent implements OnDestroy {
   get ageRangeLabel(): string {
     const { lowValue, highValue } = this;
     const prefix = 'Age: ';
-    return lowValue === highValue ? prefix + String(lowValue) : prefix + `${lowValue} - ${highValue > 89 ? '89+' : highValue}`;
+    return lowValue === highValue ? prefix +
+      String(lowValue) : prefix + `${lowValue}-${highValue > 89 ? highValue - 1 + '+' : highValue}`;
```

Comment:
Compare to options.ceil instead of the hard coded value, i.e. highValue === options.ceil ? ...